modules/apps/transfer/internal/convert/convert_test.go

```diff
@@ -0,0 +1,112 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	v1types "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	v3types "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types/v3"
+)
+
+func TestConvertPacketV1ToPacketV3(t *testing.T) {
+	const (
+		sender   = "sender"
+		receiver = "receiver"
+	)
+
+	testCases := []struct {
+		name        string
+		v1Data      v1types.FungibleTokenPacketData
+		v3Data      v3types.FungibleTokenPacketData
+		shouldPanic bool
```

Comment:
```suggestion
		expPanic bool
```
This can even be made a string/any to see if the panic message is the expected message

Reply:
couldn't we do `expPanic error` and do `RequirePanicsWithError` since there is no error return case?

Reply:
i can update this!